tests/test_skygrid/output_expected/mcmc_skygrid.errout

```diff
@@ -0,0 +1,17 @@
+Problem when computing waiting time.
+Problem when computing waiting time.
```

Comment:
Well, since the test passes, it would seem that it is indeed consistent across platforms and compilers, as ugly as it looks. :)

I've never used these popgen functions myself, but my guess is that this is no more worrying than the `Could not compute lnProb for <...>` warnings when RevBayes is trying to find a viable initial tree.